[0m03:28:45.089931 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000027FA65DEF90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000027FA75D7D90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000027FA779FED0>]}


============================== 03:28:45.094727 | ebc44d2c-d78c-4d72-85b4-363c13c61152 ==============================
[0m03:28:45.094727 [info ] [MainThread]: Running with dbt=1.9.8
[0m03:28:45.095688 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': 'C:\\Users\\DELL\\.dbt', 'log_path': 'logs', 'fail_fast': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt init netflix', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m03:28:45.116438 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse logging event dictionary. Failed to parse dir field: expected string or bytes-like object, got 'WindowsPath'.. Dictionary: {'dir': WindowsPath('C:/Users/DELL/.dbt')}
[0m03:28:45.117168 [info ] [MainThread]: Creating dbt configuration folder at 
[0m03:28:45.119570 [debug] [MainThread]: Starter project path: D:\My Projects\project\venv\Lib\site-packages\dbt\include\starter_project
[0m03:28:45.141908 [info ] [MainThread]: 
Your new dbt project "netflix" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m03:28:45.143141 [info ] [MainThread]: Setting up your profile.
[0m03:33:15.294801 [info ] [MainThread]: Profile netflix written to C:\Users\DELL\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m03:33:15.296848 [debug] [MainThread]: Command `dbt init` succeeded at 03:33:15.296617 after 270.39 seconds
[0m03:33:15.297487 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000027FA96D0FC0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000027FA8A617F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000027FA9679260>]}
[0m03:33:15.297987 [debug] [MainThread]: Flushing usage events
[0m03:33:16.477689 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m04:25:07.187943 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001B5438F6BA0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001B54487B9D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001B544A5FD90>]}


============================== 04:25:07.196260 | 0354080a-cc5a-41fd-b988-ac8521661f2a ==============================
[0m04:25:07.196260 [info ] [MainThread]: Running with dbt=1.9.8
[0m04:25:07.197255 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\DELL\\.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m04:25:07.197771 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path D:\My Projects\project\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m04:25:07.198833 [debug] [MainThread]: Command `dbt run` failed at 04:25:07.198706 after 0.17 seconds
[0m04:25:07.199117 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001B544AF0050>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001B544AACEF0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001B5448F28B0>]}
[0m04:25:07.199415 [debug] [MainThread]: Flushing usage events
[0m04:25:08.330750 [debug] [MainThread]: An error was encountered while trying to flush usage events
